src/SourceBrowser.Site/Scripts/search.js

```diff
@@ -2,11 +2,7 @@
     var path = window.location.pathname;
     var splitPath = path.split('/');
     splitPath = splitPath.filter(function (v) { return v !== '' });
-    if (splitPath.length <= 2)
-    {
-        $("#search-form").hide();
```

Comment:
It's physically gone from `_Layout.cshtml`. The form is only present in *_BrowseLayout.cshtml`